src/OpenTelemetry/Sdk.cs

```diff
@@ -29,10 +29,37 @@ namespace OpenTelemetry
     /// </summary>
     public static class Sdk
     {
-        public static readonly SuppressInstrumentationScope SuppressInstrumentation = new SuppressInstrumentationScope(false);
-
         private static readonly TimeSpan DefaultPushInterval = TimeSpan.FromSeconds(60);
 
+        /// <summary>
+        /// Gets a value indicating whether instrumentation is suppressed (disabled).
+        /// </summary>
+        public static bool SuppressInstrumentation => SuppressInstrumentationScope.IsSuppressed;
```

Comment:
There's more to the API now, but I think it is simpler to understand by decoupling the disposable object from the boolean check. Should we apply AggresiveInlining on these APIs?

Reply:
I guess `AggresiveInlining` is not going to help here :)